people.go

```diff
@@ -331,8 +332,11 @@ func (p People) ForEach(f func(int64, *Person) bool) {
 }
 
 // FindPeople returns all the people in the database or from the disk cache.
-func FindPeople(ctx context.Context, connString string, cachePath string, blacklist Blacklist) (
-	People, map[string]int, map[string]int, error) {
+func FindPeople(ctx context.Context, connString string, cachePath string, blacklist Blacklist,
+	recentMonths int) (People, map[string]*Frequency, map[string]*Frequency, error) {
+	if recentMonths == 0 {
+		recentMonths = 12
```

Comment:
"12" should be the default value in the cmdline arguments. This function should not allow 0 - this should be a panic.